source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.md

```diff
@@ -200,11 +198,28 @@ mongodb://mongodb1.mongodb.com:27317,mongodb2.mongodb.com:27107/?ssl=true&replic
 
 ## Test Plan
 
+### Spec Tests 
 See README.md in the accompanying [test directory](tests).
 
 Additionally, see the `mongodb+srv` test `invalid-uris.yml` in the
 [Connection String Spec tests](../connection-string/tests).
 
+### Prose Tests
+* Test #1: The driver should not throw an error when given a valid SRV record that only contains the name of the domain and the TLD.
```

Comment:
I think you can merge this into one test with two cases, the subject is sort of like 
- "when given an SRV" 
  - "with one domain level" 
  - "with two domain levels"
Parse the connection string without throwing a parse related error.

We have to stub DNS but other driver will be able to just call their client constructor, so I think leave out the details about setup since it will be different for each language, maybe some drivers can look for a failure to connect specifically checking that it isn't a parse related error.

Reply:
Good idea, I changed it to this format